src/contrib/testkits/Akka.TestKit.Xunit2/TestKit.cs

```diff
@@ -122,26 +125,9 @@ public TestKit(string config, ITestOutputHelper output = null)
         /// </summary>
         protected static XunitAssertions Assertions { get; } = new XunitAssertions();
 
-        /// <summary>
-        /// This method is called when a test ends.
-        ///
-        /// <remarks>
-        /// If you override this, then make sure you either call base.AfterAllAsync()
-        /// to shut down the system. Otherwise a memory leak will occur.
-        /// </remarks>
-        /// </summary>
-        protected virtual async Task AfterAllAsync()
-        {
-#pragma warning disable CS0618
-            AfterAll();
-#pragma warning restore CS0618
-            await ShutdownAsync();
-        }
-        
         /// <summary>
         /// This method is called when a test ends.
         /// </summary>
-        [Obsolete("AfterAll() is deprecated, please use AfterAllAsync() instead")]
         protected virtual void AfterAll()
         {
         }
```

Comment:
`Shutdown` needs to be called here

Reply:
no, its not, it is the cause of multiple headaches, the actor system need to be called inside Dispose, not AfterAll